### PR TITLE
chore(docs): minor updates

### PR DESCRIPTION
**Pull request by devjsc**
## Proposed Changes

_[describe the changes here...]_

## Linked Issues

_[if applicable, add links to issues resolved by this PR]_

## Types of changes

_What type of change does this pull request make (put an `x` in the boxes that apply)?_

- [ ] Content update.
- [ ] Bug fix (non-breaking change that fixes an issue).
- [ ] New feature added (non-breaking change that adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to stop working as expected).
- [ ] Something else (e.g., tests, scripts, example, deployment, infrastructure).

## Checklist

_Put an `x` in the boxes that apply:_

- [ ] I have read the [CONTRIBUTING](https://github.com/fetchai/docs/blob/main/CONTRIBUTING.md) guide
- [ ] Checks and tests pass locally

## Further comments

_[if this is a relatively large or complex change, kick off a discussion by explaining why you chose the solution you did, what alternatives you considered, etc...]_

### CODE REVIEW

File: pages/guides/agents/intermediate/langchain-rag-agent.mdx [hunk 1]
```diff
@@ -5,9 +5,9 @@ import PackageVersion from 'components/package-version'
...
 
-The process of creating RAG agents has been improved by offering a decentralized and modular framework through the uagents and ai_engine libraries. These tools streamline the integration of AI models like OpenAI and Cohere, enabling more efficient and scalable development of intelligent agents. With enhanced interoperability, security, and resource management, this framework allows developers to quickly build and deploy sophisticated agents that can effectively answer questions based on any provided document, making the entire process faster and more robust.
+The process of creating RAG agents has been improved by offering a decentralized and modular framework through the uAgents and ai_engine libraries. These tools streamline the integration of AI models like OpenAI and Cohere, enabling more efficient and scalable development of intelligent agents. With enhanced interoperability, security, and resource management, this framework allows developers to quickly build and deploy sophisticated agents that can effectively answer questions based on any provided document, making the entire process faster and more robust.
```
Comment (FelixNicolaeBucsa) [inline pages/guides/agents/intermediate/langchain-rag-agent.mdx:10]: Is it uAgents? Shouldn't we keep everything Agent?

Comment (FelixNicolaeBucsa) [reply pages/guides/agents/intermediate/langchain-rag-agent.mdx:10]: `uagents` would be the package right? Then, Agents is for the rest?

**Review by FelixNicolaeBucsa — COMMENTED** [review]
The Agents/uAgents terminology isnt completely clear to me. We should have some sort of guidelines for this so that it can be shared with everyone so that terminology is coherent across all resources in docs. A similar conversation arose in the past with Maria and Freya as far as I remember

File: pages/guides/agents/intermediate/langchain-rag-agent.mdx (hunk 1 above)
Comment (devjsc) [reply pages/guides/agents/intermediate/langchain-rag-agent.mdx:10]: We never refer to ourselves as fetch, unless in conversation verbally, therefore we are Fetch.ai
uAgents should be cased, unless it's to describe code as in `import uagents` etc.

**Pull request merged**